website/content/docs/agent-and-proxy/agent/process-supervisor.mdx

```diff
@@ -13,7 +13,8 @@ a process via environment variables using
 [Consul Template markup][consul-templating-language].
 
 -> If you are running your applications in a Kubernetes cluster, we recommend
```

Comment:
It will _probably_ work but I'm a bit concerned about using a process supervisor (which restarts another process) within Kubernetes. I think it's OK to keep it as s `instead` since it's a stronger guidance towards an establish (HashiCorp sanctioned workflow).

Reply:
If we were running this in production outside of K8S, I'd expect it would likely be managed by e.g. Systemd or similar, so I don't know that that's a huge blocker?

If I were reading this as a user, I might take from this that this this approach wouldn't work, as opposed to wording like "also consider" or "consider this first" etc. Ultimately, I don't mind having strong guidance if we want to push users to wards other solutions, though.

Reply:
Resolving this: we discussed it on Slack and realized there's no reason why it wouldn't work, and the current wording might suggest that it wouldn't work. Wording updated :)

Reply:
Had an offline discussion on this and I've been convinced :)